dependencies.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
when is this file used?

Reply:
It's supposed to be run before running `docker build`. I'll include more details in the README before the final PR is ready

Reply:
I got rid of the `dependencies.sh` in [6261f8d](https://github.com/metal-toolbox/ironlib/pull/84/commits/6261f8d0a6774aed0794150e372eb040997ce162) it was only used to fetch and install the epel RPM package so I moved that to Dockerfile.